billing/cycle.md

```diff
@@ -4,11 +4,14 @@ description: how to change yor billing cycle in Docker Hub
 keywords: billing, cycle, payments, 
 ---
 
-You can pay for a Personal, Pro, or Team subscription on a monthly or yearly billing cycle.
+You can pay for a Pro or Team subscription on a monthly or yearly billing cycle.
 
-If you currently have a monthly billing cycle, you can choose to switch to an annual billing cycle. 
+If you currently have a monthly billing cycle, you can choose to switch to an annual billing cycle. Note that you are unable to switch from an annual billing cycle to monthly. 
```

Comment:
nit: "..Note that it's not possible to switch from an annual billing cycle to monthly. "